ckanext/versioning/common.py

```diff
@@ -32,3 +32,19 @@ def create_author_from_context(context):
         author = Author(name=context['user'])
 
     return author
+
+
+def tag_to_dict(tag):
+    ''' Returns a dict version of a TagInfo object
+
+    # TODO: Refactor or move it to metastore-lib
```

Comment:
I agree this would be nicer in metastore-lib as some kind of utility or `.as_dict()` method. But I think this could be narrowed down by calling `tag_dict = tag.__dict__` and then overriding some of the fields (`created`, `author` and `author_email`)

```